types/akamai-edgeworkers/url-search-params.d.ts

```diff
@@ -0,0 +1,56 @@
+export default class URLSearchParams {
```

Comment:
Instead of using a `"paths"` mapping, you could move the contents of `url-search-params.d.ts` into `index.d.ts` as an ambient module, similar to how NodeJS handles its module definitions:

```ts
declare module "url-search-params" {
    export class URLSearchParams { ... }
    ...
}
```